resources/user-guides/index.md

```diff
@@ -35,6 +30,32 @@ title: titles.userguides
             </div>
         </div>
     </section>
+    <section class="container">
+        <div class="row">
+            <div class="left half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+                <div class="info-block">
+                    <div class="row">
+                        <div class="col">
+                            <h2>{% t user-guides.backup %}</h2>
+                            <p><a href="{{site.baseurl}}/resources/user-guides/Offline_Backup.html">{% t user-guides.offline-backup %}</a></p>
```

Comment:
Do we expect more back up guides to be added? Otherwise this section will stay with one entry for a very long time. Maybe better to squash it into "General"?

Reply:
Yes, I have in mind at least 2-3 more guides. I think this topic is important and should have it's own category.